server/src/main/scala/org/http4s/server/package.scala

```diff
@@ -52,7 +52,7 @@ package object server {
 
   object ServerRequestKeys {
     val SecureSession: Key[Option[SecureSession]] =
-      Key.newKey[IO, Option[SecureSession]].unsafeRunSync()
+      Key.newKey[SyncIO, Option[SecureSession]].unsafeRunSync()
```

Comment:
We perhaps should also have used this in cats-effect-2, but probably not worth going back to correct.